src/privatesend/privatesend-client.cpp

```diff
@@ -683,6 +683,11 @@ void CPrivateSendClientManager::AddSkippedDenom(const CAmount& nDenomValue)
     vecDenominationsSkipped.push_back(nDenomValue);
 }
 
+void CPrivateSendClientManager::RemoveSkippedDenom(const CAmount& nDenomValue)
+{
+    std::remove(vecDenominationsSkipped.begin(), vecDenominationsSkipped.end(), nDenomValue);
```

Comment:
```suggestion
    vecDenominationsSkipped.erase(std::remove(vecDenominationsSkipped.begin(), vecDenominationsSkipped.end(), nDenomValue), vecDenominationsSkipped.end());
```
Welcome to the world of STL strangeness ;)
std::remove doesn't really "remove" stuff, it moves elements from the right to the left when an element needs to be "removed", but never actually deletes the elements. Instead, it returns the iterator for the element after the last moved one, giving you a hint on where to delete by yourself.

Reply:
That's crazy, missed that part in docs. Thanks!